File: uri 1008.java
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args)  {
 
        Scanner sc = new Scanner (System.in);
        int n,h;
        double m,salary;
        n = sc.nextInt();
        h = sc.nextInt();
        m = sc.nextDouble();
        salary = h*m;
        System.out.println("NUMBER = "+n);
        System.out.printf("SALARY = U$ %.2f\n",salary);
 
    }
 
}

File: uri 1008.c

#include <stdio.h>

int main() {

    int A,NUMBER;
    double B,C, SALARY;
    scanf("%d%lf%lf",&A,&B,&C);
    NUMBER = A;
    SALARY = B*C;
    printf("NUMBER = %d\n",NUMBER);
    printf("SALARY = U$ %0.2lf",SALARY);

    return 0;
}
